integration-tests/ast/if_else.mj.stdout

```diff
@@ -0,0 +1,13 @@
+class IfElse {
+	public static void main(String[] args) {
+		if (true)
+			return;
```

Comment:
why no braces around the `return` ( as well as in line 7 ?

Reply:
This is correct. In Java the braces are optional if the `if` body consists of a single statement.

Or is there a specification, that says that `if` bodies always have to be parenthesized?

Reply:
I just wonder how we distinguish this one comparated to the two single-statement `if` / `else` variants in the `hello_world.mj`. WIll make a comment there...